WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/download/usecases/PostBackupDownloadUseCase.kt

```diff
@@ -0,0 +1,54 @@
+package org.wordpress.android.ui.jetpack.backup.download.usecases
+
+import kotlinx.coroutines.flow.flow
+import org.wordpress.android.R
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.fluxc.store.ActivityLogStore
+import org.wordpress.android.fluxc.store.ActivityLogStore.BackupDownloadPayload
+import org.wordpress.android.fluxc.store.ActivityLogStore.BackupDownloadRequestTypes
+import org.wordpress.android.ui.jetpack.backup.download.handlers.BackupDownloadHandler.BackupDownloadHandlerStatus
+import org.wordpress.android.ui.utils.UiString
+import org.wordpress.android.ui.utils.UiString.UiStringRes
+import org.wordpress.android.ui.utils.UiString.UiStringText
+import org.wordpress.android.util.NetworkUtilsWrapper
+import javax.inject.Inject
+
+class PostBackupDownloadUseCase @Inject constructor(
+    private val networkUtilsWrapper: NetworkUtilsWrapper,
+    private val activityLogStore: ActivityLogStore
+) {
+    suspend fun postBackupDownloadRequest(
+        rewindId: String,
+        site: SiteModel,
+        types: BackupDownloadRequestTypes
+    ) = flow {
```

Comment:
Same as above - Should we move the ui related things (UiStrings) to the above layer. Wdyt?

Reply:
Yes, moving them